tests/static_data/single_publication/layers_test.py

```diff
@@ -332,4 +332,6 @@ def test_layer_style_external_images(workspace, publ_type, publication):
     ensure_publication(workspace, publ_type, publication)
 
     for image_file in data.PUBLICATIONS[(workspace, publ_type, publication)][data.TEST_DATA]['external_images']:
-        process_client.get_workspace_layer_style_external_image(workspace, publication, image_file)
+        response = process_client.get_workspace_layer_style_external_image(workspace, publication, image_file)
+        assert response.headers['Content-Type'] == 'image/svg'
+        assert len(response.content) > 0, response.content
```

Comment:
Test that response is the same as expected file. Add "expected file" to dictionary